tests/rgw/sanity_rgw.py

```diff
@@ -187,14 +187,16 @@ def run(ceph_cluster, **kw):
 
     if run_io_verify:
         log.info("running io verify script")
-        verify_out, err = exec_from.exec_command(
+        verify_status = exec_from.exec_command(
             cmd=f"sudo {python_cmd} "
             + test_folder_path
             + lib_dir
             + f"read_io_info.py -c {config_file_name}",
             timeout=timeout,
+            long_running=True,
         )
-        log.info(verify_out)
-        log.error(err)
+        log.info(f"verify io status code is : {verify_status}")
+        if verify_status != 0:
+            return verify_status
```

Comment:
Please add error message before returning status code

Reply:
@udaysk23 , added log.error(verify_status) 
yes @sunilkumarn417 , it returns exit code only